generator/src/lib.rs

```diff
@@ -1075,13 +1075,36 @@ pub fn generate_extension_commands<'a>(
     let fp = generate_function_pointers(ident, &commands, &aliases, fn_cache);
     let byte_name = format!("{}\0", extension_name);
 
+    let spec_version = items
+        .iter()
+        .find_map(|ext_item| match ext_item {
+            vk_parse::ExtensionChild::Require { items, .. } => {
+                items.iter().find_map(|item| match item {
+                    vk_parse::InterfaceItem::Enum(ref e) if e.name.contains("SPEC_VERSION") => {
+                        Some(e)
+                    }
+                    _ => None,
+                })
+            }
+            _ => None,
+        })
+        .and_then(|e| {
+            if let vk_parse::EnumSpec::Value { value, .. } = &e.spec {
+                let v: u32 = str::parse(value).unwrap();
+                Some(quote!(pub const SPEC_VERSION: u32 = #v;))
+            } else {
+                None
+            }
+        });
```

Comment:
How about collapsing the `and_then` case into `find_map`?

Reply:
I think this is fine

Reply:
Saves on nesting, that's worth something tool and might make the filter reusable in the future.